javascript-loops/main.js

```diff
@@ -0,0 +1,56 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  for (var currentNumber = 1; currentNumber < 10; currentNumber++);
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  for (var currentNumber = 2; currentNumber < 20; currentNumber += 2);
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var repeated = '';
+  for (var count = 1; count < times; count++);
```

Comment:
This needs to be a `while` loop, and also you have no code block for the loop, so nothing is being done on each iteration.